### DISCUSSION TITLE
How to access component-specific configurations parameters?

### DISCUSSION THREAD

**Discussion by outsch** [joomengine/Joomla-Component-Builder#799] [Q&A]
May be again a very basic question, but how can I access parameters which I added to my component in the code of e.g. a site view?
I have tried testing with the following code in the tmpl/default.php of the respective view:

```
$myParams = $this->app->getParams();
<p>$myParams are:<?php print_r($myParams) ?>
```

But I only get the result:
`$myComponent  parameters: Warning: Undefined variable $myParams`

And would the name of a config parameter set in the component config be drived from the field name that I used or how else?
Thanaks in advance for any hint.

**Comment by kmd-muc** [comment]
try: 
`$params	= JComponentHelper::getParams('com_[[[component]]]');`
and then:
`$my_param = $params->get('my_param');`
